Sources/SwiftSyntaxBuilder/gyb_generated/BuildableNodes.swift

```diff
@@ -12834,20 +12512,6 @@ public struct GenericArgumentClause: SyntaxBuildable, ExpressibleAsGenericArgume
     assert(rightAngleBracket.text == ">")
   }
 
-  /// A convenience initializer that allows:
-  ///  - Initializing syntax collections using result builders
-  ///  - Initializing tokens without default text using strings
-  public init(
-    leftAngleBracket: TokenSyntax = TokenSyntax.`leftAngle`,
-    rightAngleBracket: TokenSyntax = TokenSyntax.`rightAngle`,
-    @GenericArgumentListBuilder argumentsBuilder: () -> ExpressibleAsGenericArgumentList = { GenericArgumentList([]) }
```

Comment:
Please keep this as `GenericArgument` conforms to `HasTrailingComma`

Reply:
✅ I think result builder would make sense here because there’s no reason to assume that types only have a single generic argument.